src/main/java/org/snomed/snowstorm/mrcm/MRCMLoader.java

```diff
@@ -1,6 +1,9 @@
 package org.snomed.snowstorm.mrcm;
```

Comment:
In MRCMLoader, I have moved the bulk of the logic into private methods. As well as this, I am storing the MRCM object in a map. Subsequent requests will use the MRCM object from the map if the same branchPath is used.

Reply:
I have added a new method to MRCMLoader; this new method is the method that has the cache. The older method remains the same.